ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -111,16 +109,20 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
   }
 
   toggleDisability(value) {
```

Comment:
Typing nit: it's never too late to start incorporating some type-safety here, e.g. value --> value: Boolean

Reply:
Will do, if I don't get rid of these methods.